server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/SpamAssassinTest.java

```diff
@@ -56,8 +57,8 @@
 public class SpamAssassinTest {
     private static final String SPAM_CONTENT = "XJS*C4JDBQADN1.NSBN3*2IDNEN*GTUBE-STANDARD-ANTI-UBE-TEST-EMAIL*C.34X";
 
-    @Rule
-    public SwarmGenericContainer spamAssassinContainer = new SwarmGenericContainer(Images.SPAMASSASSIN)
+    @ClassRule
+    public static SwarmGenericContainer spamAssassinContainer = new SwarmGenericContainer(Images.SPAMASSASSIN)
```

Comment:
do we need to clean up `spamAssassinContainer` after each test?

Reply:
No we don't.